src/lib/openapi/spec/health-overview-schema.ts

```diff
@@ -23,6 +23,7 @@ export const healthOverviewSchema = {
         },
         description: {
             type: 'string',
+            nullable: true,
```

Comment:
I guess this is different than not being required. Right?

Reply:
this is when description is there and set to null. We were getting those values in the value space, so adding them to the type space